packages/workbox-core/models/messages/messages.mjs

```diff
@@ -93,4 +104,25 @@ export default {
       `setCacheDeatils({${cacheNameId}: '...'}). Received a value of ` +
       `'${JSON.stringify(value)}'`;
   },
+
+  'unregister-routes-no-routes-found-for-method': ({method}) => {
```

Comment:
This error's code is a little confusing.

`unregister-route-but-not-found-with-method` would be better IMO.